k8s.tf

```diff
@@ -10,6 +10,15 @@ locals {
   actuator_port = 18000
   wd_port       = 48869
 }
+
+resource "kubernetes_service_account" "waggle_dance" {
+  metadata {
+    name      = local.instance_alias
+    namespace = var.k8s_namespace
```

Comment:
In `apiary-data-lake`, and all other places I've seen you add IRSA support, you also added a section like:
```
    annotations = {
      "eks.amazonaws.com/role-arn" = var.oidc_provider == "" ? "" : aws_iam_role.apiary_hms_readonly.arn
    }
```
and then setup the `oidc_provider` variable.  Is that not necessary here?